### ISSUE TITLE
Replace Delhi, Jaipur, and Kolkata Buttons with Most Recent Three Searches

### ISSUE THREAD

**Issue by adaharshsingh** [adaharshsingh/Propacity#34]
The current UI displays static buttons for Delhi, Jaipur, and Kolkata. 
To improve the user experience, I propose replacing these buttons with dynamic ones that show the most recent three searches made by the user.

### Requirements:

- Track the user's most recent three searches (e.g., city names or locations).
- Dynamically update the buttons to display these recent searches.

### Expected Behavior:

- Each time a user searches for a city or location, it should be added to the list of recent searches.
- The buttons should reflect the most recent three searches, replacing the static ones.
- This will allow users to quickly access their recent searches without having to re-enter the location names.

**Comment by adaharshsingh** [comment]
Go Ahead!

**Comment by Thejaggeddevil** [comment]
I'd like to work on this issue

**Comment by adaharshsingh** [comment]
Thank you for your contribution! Your code has resolved the issue mentioned here, and I have merged it. However, I noticed a bug in the implementation, which I've documented in issue #38 .

Additionally, the code doesn’t seem to update the recent searches for some cities, like Bhopal. I'm not sure why this is happening. If you're interested in addressing these issues, send the PR.

Thanks again for your valuable contribution!
